Source/Core/Core/Movie.cpp

```diff
@@ -1580,4 +1580,77 @@ void Shutdown()
   tmpInput = nullptr;
   tmpInputAllocated = 0;
 }
+
+void CheckWiiSaves()
+{
+  std::string save_path = Common::GetTitleDataPath(GetTitleId(), Common::FROM_SESSION_ROOT);
+  std::string user_save_path = Common::GetTitleDataPath(GetTitleId(), Common::FROM_CONFIGURED_ROOT);
+  if (IsRecordingInput())
+  {
+    // TODO: Check for the actual save data
+    SetClearSave(!File::Exists(user_save_path + "banner.bin"));
+  }
+
+  if (Core::g_want_determinism && !IsStartingFromClearSave())
+  {
+    // Copy the current user's save to the Blank NAND
+    if (File::Exists(user_save_path + "banner.bin"))
+    {
+      File::CopyDir(user_save_path, save_path);
+    }
+  }
+}
+
+static std::string s_temp_wii_root;
+
+void InitializeWiiRoot(bool use_dummy)
```

Comment:
InitializeWiiRoot and ShutdownWiiRoot are already in WiiRoot.cpp. Don't create duplicates.